src/main/java/core/basesyntax/dao/CsvFileReaderImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Operation;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CsvFileReaderImpl implements CsvFileReader {
+
```

Comment:
be consistent about new line after class declaration, in previous file you didn't have this line